app/containers/AddJob.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
Use `createClass` for this component, and let the `dispatch` happen through `mapActionsToProps` (you may need to tweak this a little, as I just wrote this blindly :):

```javascript
import React, { createClass } from 'react';
import { connect } from 'react-redux';
import { addJob } from '../actions/DashboardActions';

const AddJob = createClass({
  displayName: 'AddJob',

  onSubmit(event) {
    event.preventDefault();
    this.props.addJob({
      companyName: this.companyNameInput.value,
      status: this.jobStatusInput.value
    });
    this.companyNameInput.value = '';
    this.jobStatusInput.value = '';
  },

  render() {
    return (
      <form onSubmit={this.onSubmit}>
        <input
          ref={nameInput => { this.companyNameInput = nameInput }}
          placeholder={'Company Name'}
        />
        <input
          ref={statusInput => { this.jobStatusInput = statusInput }}
          placeholder={'Status'}
        />
        <button type="submit">
          Add Job
        </button>
      </form>
    );
  }
});

const mapActionsToProps = {
  onAddJob: addJob
};

export default connect(null, mapActionsToProps)(AddJob);
```